internal/compiler/llr/lower_expression.rs

```diff
@@ -644,8 +644,16 @@ fn box_layout_data(
         })
     };
 
-    let repeater_count =
-        layout.elems.iter().filter(|i| i.element.borrow().repeated.is_some()).count();
+    let repeater_count = layout
+        .elems
+        .iter()
+        .filter(|i| {
+            !matches!(
+                i.element.borrow().repeated,
+                Some(crate::object_tree::RepeatedElementInfo::Embedding(_))
+            )
```

Comment:
Another pattern that might shorten this is to have an `fn as_repeater(&self) -> Option<&RepeaterInfo>` along with `fn as_embed_element(&self) -> Option<&EmbeddingInfo>`. Then you can just use `is_some()` here and don't have to repeat the long match.